src/config/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const SAFE_APPS_URL_PROD = 'https://apps.gnosis-safe.io'
```

Comment:
What was your goal when you moved all urls to one file? 

I feel like this is an unnecessary change. Now when you want to understand what endpoint an environment uses, you will have to go to a config file, then see what constant is used, then go to constants file and get the value. I think doing it this way loses the point of config files - having configuration in one file

Reply:
Do you think it is better to replicate the URLs in more than one place? if one changes, you have to search everywhere and replicate the value. Also, you don't have a centralized place to see all the endpoints we are using. 

To check the value you can ctrl+click in the editor and it navigates you to the value. Or even better, hovering the ENV will show you the value.
![image](https://user-images.githubusercontent.com/4376509/83946339-7413e900-a7e6-11ea-808e-e8966c0527c8.png)

But if you want, I can get rid the file.

Reply:
> Do you think it is better to replicate the URLs in more than one place? if one of them changes, you have to search everywhere and replicate the value

When it comes to DevOps thing I would be better explicitly changing the url for each env so I know what environments will be affected by my changes. The amount of maximum replications is 2, so not too much overhead.

> Also, you don't have a centralized place to see all the endpoints we are using.

I agree, in general we are missing describing many things we're using or doing, but this would be better in a form of documentation. Maybe we should start a [wiki](https://github.com/gnosis/safe-react/wiki)?

> To check the value you can ctrl+click in the editor and it navigates you to the value. Or even better, hovering the ENV will show you the value.

Yep I know about this thing, but you are not always inside the editor, you can use github to quickly edit the file or something else that doesn't have this feature.

Reply:
yes, that's right too. @fernandomg @Agupane wdyt?